lib/main.dart

```diff
@@ -1,80 +1,61 @@
-import 'package:e_wallet/models/User.dart';
-import 'package:e_wallet/providers/UserProvider.dart';
+import 'package:e_wallet/models/user_model.dart';
```

Comment:
Organize imports by sections and alphabetize sections. [Please see this reference](https://dart.dev/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports)

You may also use Android studio's `Format code on save` under Languages & Framework > Flutter > Editor
![image](https://user-images.githubusercontent.com/60649968/104160904-dee86b00-542c-11eb-9bb4-5b7fe9507346.png)

Reply:
I'll keep it in mind. It's a handy setting.